snippets/wasm/webhook.ts

```diff
@@ -0,0 +1,21 @@
+import { type BindingLiquidSdk } from '@breeztech/breez-sdk-liquid'
+
+const _registerWebhook = async (sdk: BindingLiquidSdk) => {
+  // ANCHOR: register-webook
+  try {
+    await sdk.registerWebhook('https://your-nds-service.com/notify?platform=ios&token=<PUSH_TOKEN>')
+  } catch (err) {
+    console.error(err)
+  }
+  // ANCHOR_END: register-webook
+}
+
+const _unregisterWebhook = async (sdk: BindingLiquidSdk) => {
+  // ANCHOR: unregister-webook
+  try {
+    await sdk.unregisterWebhook()
+  } catch (err) {
+    console.error(err)
+  }
+  // ANCHOR_END: unregister-webook
```

Comment:
The corresponding references in `using_webhooks.md` are missing

Reply:
Fixed [ec08971](https://github.com/breez/breez-sdk-liquid-docs/pull/92/commits/ec08971e083f667fc30b1b321a3daa657da4c0db)